File: Examples/src/main/java/com/aspose/words/examples/rendering_printing/SaveAsMultipageTiff.java
package com.aspose.words.examples.rendering_printing;

import com.aspose.words.*;
import com.aspose.words.examples.Utils;

public class SaveAsMultipageTiff {

    private static final String dataDir = Utils.getSharedDataDir(SaveAsMultipageTiff.class) + "RenderingAndPrinting/";

    public static void main(String[] args) throws Exception {

        // Open the document.
        Document doc = new Document(dataDir + "TestFile.MultipageTIFF.docx");
        //ExStart:SaveAsTIFF
        // Save the document as multipage TIFF.
        doc.save(dataDir + "TestFile.MultipageTIFF_out.tiff");
        //ExEnd:SaveAsTIFF

        // ExStart:SaveAsTIFFUsingImageSaveOptions
        // Create an ImageSaveOptions object to pass to the Save method
        ImageSaveOptions options = new ImageSaveOptions(SaveFormat.TIFF);
        options.setPageSet(new PageSet(0, 2));
        options.setTiffCompression(TiffCompression.CCITT_4);
        options.setResolution(160);
        doc.save(dataDir + "TestFileWithOptions_Out.tiff", options);
    	// ExEnd:SaveAsTIFFUsingImageSaveOptions
        System.out.println("Document saved as multi page TIFF successfully.");
    }
}

File: Plugins/Aspose_Words_Java_for_Jython/asposewords/rendering_printing/SaveAsMultipageTiff.py
from asposewords import Settings
from com.aspose.words import Document
from com.aspose.words import ImageSaveOptions
from com.aspose.words import SaveFormat
from com.aspose.words import TiffCompression

class SaveAsMultipageTiff:

    def __init__(self):
        dataDir = Settings.dataDir + 'rendering_printing/'
            
        # Open the document.
        doc = Document(dataDir + "TestFile.doc")

        # Save the document as multipage TIFF.
        doc.save(dataDir + "TestFile Out.tiff")
        
        # Create an ImageSaveOptions object to pass to the Save method
        options = ImageSaveOptions(SaveFormat.TIFF)
        options.setPageIndex(0)
        options.setPageCount(2)
        options.setTiffCompression(TiffCompression.CCITT_4)
        options.setResolution(160)

        doc.save(dataDir + "TestFileWithOptions Out.tiff", options)

        "Document saved as multi page TIFF successfully."

if __name__ == '__main__':
    SaveAsMultipageTiff()